tests/test_formatting.py

```diff
@@ -335,3 +336,14 @@ def test_formatting_with_options_metavar_empty(runner):
     cli = click.Command("cli", options_metavar="", params=[click.Argument(["var"])])
     result = runner.invoke(cli, ["--help"])
     assert "Usage: cli VAR\n" in result.output
+
+
+def test_help_formatter_write_text():
+    # Failed because of https://github.com/pallets/click/issues/1871
```

Comment:
In general we don't leave comments pointing to issues. No need for a comment here, but if you want to write one just describe what's being tested succinctly and directly.

Reply:
I removed the comment.